src/SFA.DAS.EAS.Web/FileDownloads/Payments/CsvTransactionFormatter.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using SFA.DAS.EAS.Domain.Models.Transaction;
+using SFA.DAS.EAS.Web.ViewModels;
+
+namespace SFA.DAS.EAS.Web.FileDownloads.Payments
+{
+    public class CsvTransactionFormatter : ITransactionFormatter
+    {
+        public string MimeType => "text/csv";
+
+        public string FileExtension => "csv";
+
+        public DownloadFormatType DownloadFormatType =>
+            DownloadFormatType.Csv;
+
+        public byte[] GetFileData(List<TransactionDownloadLine> transactions)
+        {
+            var builder = new StringBuilder();
+
+            WriteHeaderCsv(builder);
+
+            WriteRowsCsv(transactions, builder);
+			
+			var csvContent = builder.ToString();
+
+            return Encoding.UTF8.GetBytes(csvContent);
+        }
+
+        private static void WriteHeaderCsv(StringBuilder builder)
+        {
+            builder.AppendFormat("Transaction date,");
+            builder.AppendFormat("Transaction type,");
+            builder.AppendFormat("PAYE scheme,");
+            builder.AppendFormat("Payroll month,");
+            builder.AppendFormat("Levy declared,");
+            builder.AppendFormat("English %,");
+            builder.AppendFormat("10% top up,");
+            builder.AppendFormat("Training provider,");
+            builder.AppendFormat("Cohort reference,");
+            builder.AppendFormat("Unique learner number,");
+            builder.AppendFormat("Apprentice,");
+            builder.AppendFormat("Apprenticeship training course,");
+            builder.AppendFormat("Paid from levy,");
+            builder.AppendFormat("Your contribution,");
+            builder.AppendFormat("Government contribution,");
+            builder.AppendFormat("Total,");
+            builder.AppendFormat(Environment.NewLine);
+        }
+
+        private static void WriteRowsCsv(List<TransactionDownloadLine> transactions, StringBuilder builder)
+        {
+            foreach (var transaction in transactions)
+            {
+                builder.AppendFormat("{0},", transaction.DateCreated);
+                builder.AppendFormat("{0},", transaction.TransactionType);
```

Comment:
Is string interpolation required here?  If not then could just use simpler Append(...) be used instead?

Reply:
done